packages/resource-scheduler/src/components/SearchPanel/SearchPanel.js

```diff
@@ -1,80 +1,63 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import EntityListApp from 'tocco-entity-list/src/main'
-import Panel from 'react-bootstrap/lib/Panel'
-import PanelGroup from 'react-bootstrap/lib/PanelGroup'
 import {
   design,
   Icon,
+  Panel,
   Typography
 } from 'tocco-ui'
 
-import {StyledPanel} from './StyledSearchPanel'
-
 class SearchPanel extends React.PureComponent {
   constructor() {
     super()
 
     this.state = {activeKey: -1}
   }
 
-  handleAccordionSelect = activeKey => {
-    this.setState({activeKey: this.state.activeKey === activeKey ? -1 : activeKey})
-  }
-
   handleSelect = name => selection => {
     this.props.updateRequestedCalendars(name, selection)
   }
 
-  getSearchLists = calendarTypes => {
-    const accordionHeader = (label, name, color) =>
-      <div onClick={() => { this.handleAccordionSelect(name) }}>
+  accordionHeader = (label, color) =>
```

Comment:
make AccordionHeader a stateless react component rather than a function that returns jsx